java/src/jmri/implementation/AbstractTurnout.java

```diff
@@ -189,7 +189,7 @@ public void newKnownState(int s) {
     /**
      * Show whether state is one you can safely run trains over.
      *
-     * @return true iff state is a valid one and the known state is the same as
+     * @return true if state is a valid one and the known state is the same as
```

Comment:
Iff is not a typo here, so please do not change this documentation, as iff != if

Reply:
As Randal said, iff is not a typo.  It says "if and only if" and is correct usage of mathematical phrasing

Reply:
Although I use 'iff' a lot (and may have even written that comment), I agree that it can be unclear mostly because it's easy to miss.  And in this case, it's redundant: If the @return clause is written properly, then it already is describing _all_ the times that true is returned, so the "only if" isn't needed. 

A quick scan (with whitespace assumptions) find a half-dozen uses in @return, _one_ other use in the source (which might actually be a valid one) and one use on the Create Decoder Advanced doc page.  

I'm happy to live without these.